src/emails/getEmailL10n.ts

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { SubscriberRow } from "knex/types/tables";
+import { acceptedLanguages, negotiateLanguages } from "@fluent/langneg";
+import { ExtendedReactLocalization } from "../app/hooks/l10n";
+import { getSpecificL10nSync } from "../app/functions/server/mockL10n";
+
+export function getEmailL10n(
+  subscriber: SubscriberRow,
```

Comment:
Not critical, but I wonder if instead of using `SubscriberRow` we should have more constrained types - e.g. we could `Pick` as a way to allow-list. There are properties in `SubscriberRow` we would never want to appear in an email.

Reply:
That's a great point. I've implemented that in the followup PR at https://github.com/mozilla/blurts-server/pull/4370.